src/data/multidim/multidimindex.h

```diff
@@ -20,14 +20,14 @@ static inline std::string to_string(const MultiIndex &multiIndex)
 {
 	typedef Text::SmartString S;
 	return "[ "
-	     + S::join<std::vector, std::string>(
+	     + S::join(
 	         S::map(multiIndex,
 	             [](const Index &index)
 	             {
 		             return std::to_string(
 		                 static_cast<size_t>(index));
 	             }),
-	         std::string(", "))
+	         ", ")
```

Comment:
We don't really need the spaces in the result. join could be used with the default separator.

Reply:
Done